wallet/src/App.vue

```diff
@@ -47,12 +59,12 @@ import {
 } from "@algo-builder/web";
 import { convertMicroAlgoToAlgo } from "./utility";
 declare var AlgoSigner: any; // eslint-disable-line
```

Comment:
Could we use [algo-signer-type](https://github.com/scale-it/algo-builder/blob/develop/packages/web/src/algo-signer-types.ts) for AlgoSigner?

Reply:
No. We can't. When you have the AlgoSigner extension installed, then the AlgoSigner object is present in the windows object. I have also checked with the AlgoSigner community. It has to be `Any`. This is also described in their documentation as well.

Reply:
Okay ~~ thank!